### DISCUSSION TITLE
I keep getting microstutters

### DISCUSSION THREAD

**Discussion by nyannoying1337** [ppy/osu#18281] [Q&A] [answered]
While playing i keep getting stutters. while looking at the performance overlay the SwapBuffer seems to always stick out.
I tried a Fresh Windows install, different nvidia drivers and closed any program in the background

Windows 11 Pro, i5-10400f, Nvidia GTX 1050 Ti, 16GB RAM

![image](https://user-images.githubusercontent.com/17437302/168479056-9ab98884-0128-4d7d-a444-9b8f0047119c.png)

https://user-images.githubusercontent.com/17437302/168479497-d91cd79a-c9d0-4cd0-9147-412736efa278.mp4

[network.log](https://github.com/ppy/osu/files/8695214/network.log)
[performance.log](https://github.com/ppy/osu/files/8695215/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8695216/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8695217/updater.log)
[database.log](https://github.com/ppy/osu/files/8695218/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/8695219/legacy-ipc.log)

**Comment by peppy** [comment, marked as answer]
Red stutters imply work elsewhere on the system from GPU usage. Usually an overlay (ie. discord) rendering into the game. Sometimes something on a secondary screen. Not osu!'s end, though.

https://osu.ppy.sh/help/wiki/Performance_Troubleshooting

**Reply by nyannoying1337** [reply]
ive been testing with various settings, meaning not using Discord or Shadowplay or any other application which would interact with osu. while i understand that it might not be osu!s fault im not quite sure what else would cause it. Windows 10 also shows the same issue. on stable / other games its not been an issue. Is there a way to figure to get deeper into for me ( as user ? )

**Reply by peppy** [reply]
We can't offer user hardware troubleshooting sorry.

Please read the linked page carefully. It solves issues for 99% of people.

We will also offer direct x rendering in the future, which is known to be less susceptible to other apps causing issues.
